salt/elife-dashboard/init.sls

```diff
@@ -209,8 +199,11 @@ app-done:
 # TODO: this is an upstart service. how does it relate to daemons.conf in processes.sls
 elife-dashboard-process-queue-daemon:
     file.managed:
+        # see: /srv/elife-dashboard/process_dashboard_queue.sh
         - name: /etc/init/elife-dashboard-process-queue-daemon.conf
         - source: salt://elife-dashboard/config/etc-init-elife-dashboard-process-queue-daemon.conf
         - template: jinja
         - require:
             - cmd: app-done
+        - watch:
```

Comment:
this is a `file.managed` state so what additional behavior should it perform when watching another one?
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html doesn't say anything about it.

Reply:
Maybe https://github.com/elifesciences/elife-dashboard-formula/blob/master/salt/elife-dashboard/processes.sls#L50 is what you are after?

Reply:
Tricky to use it there too, sometimes I just resort to `cmd.run` that is always executed.

Reply:
> this is a file.managed state

oh yeah ... derp. I'll take another look.

Reply:
> Tricky to use it there too, sometimes I just resort to cmd.run that is always executed

why tricky? I'm worried I've overlooked something now ;) that was exactly what I was after, thank you. 

the problem with `cmd.run` is that the command would be run daily incurring just a little bit of downtime as whatever is stopped and started, assuming you can safely stop and start the service with the one command. I'm thinking of the docker container `cmd.run` states I'm seeing in `digests` right now. Lots of people don't like systemd, but it's services are growing on me. I liked upstart too.